compiler/ballerina-lang/src/main/java/io/ballerina/projects/ModuleDescriptor.java

```diff
@@ -65,7 +65,7 @@ public ModuleName name() {
         return moduleName;
     }
 
-    PackageID moduleCompilationId() {
+    public PackageID moduleCompilationId() {
```

Comment:
Isn't this problematic? This basically exposes an internal element through one of the public API elements. @azinneera thoughts?

Reply:
Reverted